docs/concepts/editions-overview.md

```diff
@@ -6,11 +6,11 @@
 
 .NET Core is a cloud-optimized, cross-platform implementation of the .NET Platform. It currently supports three main operating systems: Linux, Windows and OS X.
 
-There are several characteristics of .NET Core:
+The following are main characteristics of .NET Core:
```

Comment:
When I see this list, it almost makes me thing it's a list of advantages over .NET Framework. Is there anything else that we should list if I'm considering switching to .NET Core coming from other platforms? Choice of languages? ASP.NET v5/MVC? Etc.